src/main.rs

```diff
@@ -19,4 +37,20 @@ pub fn main<G: Html>() -> PerseusApp<G> {
         .template(crate::templates::index::get_template())
         .template(crate::templates::about::get_template())
         .error_views(crate::error_views::get_error_views())
+        .index_view(|context| {
+            let mysql_client = MySqlClient::new(&config.database);
+            let course_store = CourseStore::new(mysql_client);
+            prelude::provide_context(context, course_store);
+
+            sycamore::view! { context,
+                head {}
+                body() {
+                    // This creates an element into which our app will be interpolated
+                    // This uses a few tricks internally beyond the classic `<div id="root">`, so we use this wrapper for convenience
+                    PerseusRoot()
```

Comment:
Do we know if the same context is automatically passed into all other components if we add the `CourseStore` object into `context` within this scope?